docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,230 @@
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The total battery usage for appid [%s] over last 24 hrs is [%d]", app_id, battery_usage);
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	duration_val = BATTERY_MONITOR_DURATION_TYPE_1WEEK;
+	error_code = battery_monitor_get_total_usage_by_app_id(app_id, duration_val, &battery_usage);
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("The total battery usage for appid [%s] over last 7 days is [%d]", app_id, battery_usage);
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	return;
+   }
+   ```
+3. Fetching Battery Usage values for all the resources used by an Application Id for a certain duration-
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, for example:
+
+   ```
+   #include <battery_monitor.h>
+
+   void
+   func(void)
+   {
+	battery_monitor_h data_handle = NULL;
+
+	char* app_id = "org.tizen.samsung";
+	int error_code = BATTERY_MONITOR_ERROR_NONE;
+	int battery_usage = -1;
+	battery_monitor_duration_type_e duration_val = BATTERY_MONITOR_DURATION_TYPE_1DAY;
+
+	/*First use this API to fetch the usage for all the available resources in the data handle*/
+	error_code = battery_monitor_get_usage_by_app_id_for_all_resource_id(app_id, duration_val, &data_handle);
+
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("Data Handle information received");
+	else
+		printf("Error Occurred [%d]", error_code);
+
+	/*Now retrieving the information one by one from the data_handle*/
+	battery_monitor_resource_id_e resource_id = BATTERY_MONITOR_RESOURCE_ID_DISPLAY;
+
+	error_code = battery_monitor_get_usage_for_resource_id(data_handle, resource_id, &battery_usage);
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("Battery Usage by display is [%d]", battery_usage);
+
+	resource_id = BATTERY_MONITOR_RESOURCE_ID_WIFI;
+
+	error_code = battery_monitor_get_usage_for_resource_id(data_handle, resource_id, &battery_usage)
+	if (error_code == BATTERY_MONITOR_ERROR_NONE)
+		printf("Battery Usage by display is [%d]", battery_usage);
+
+	/* Similarly for other resources...
+	.
+	.
+	.
+	*/
+
+	error_code = battery_monitor_destroy(data_handle);
+	if (error_code != BATTERY_MONITOR_ERROR_NONE)
+		printf("Error Occurred [%d]", error_code);
+
+	free(app_id);
+
+	return;
+   }
+   ```
+
+<a name="resourceusage_get"></a>
+## Getting Battery usage information for a particular resource
+
+Some Battery Usage information contain combination of Resource Id and duration.
+
+1. Fetching Battery Usage values for a particular resource over certain duration of time.
+   The `battery_monitor_resource_id_e`, `battery_monitor_duration_type_e` enums can be used to provide information related to specific resource and particular duration type respectively, for example:
```

Comment:
The `battery_monitor_resource_id_e` and the `battery_monitor_duration_type_e` enums can be used to provide information related to a specific resource and a particular duration type respectively, as shown in the following code: